### PR TITLE
:zap: fix: add esha98

### LINKED ISSUE #1687: [PROFILE ADDITION] <Esha>
### Profile screenshot

Screenshot 2023-01-28 at 12 36 18 pm

### PR DESCRIPTION
**Pull request by Esha98**
resolves #1687

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/esha98.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/60232135/215237852-c021e76d-316d-4fa0-96dd-59d9666bc6fe.png",
+    "issueId": 1687,
+    "name": "Esha",
+    "username": "Esha98"
+  }
+
```
Comment (Panquesito7) [inline profiles/esha98.json:7]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

File: profiles/esha98.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/60232135/215237852-c021e76d-316d-4fa0-96dd-59d9666bc6fe.png",
```
Comment (Panquesito7) [inline profiles/esha98.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/215367152-fbb4084f-79a4-470b-a96a-1656f7a735dc.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/esha98.json (hunk 2 above)
Comment (Esha98) [reply profiles/esha98.json]: Yes, this looks decent. I used your recommended image @Panquesito7

File: profiles/esha98.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/51391473/215367152-fbb4084f-79a4-470b-a96a-1656f7a735dc.png",
+    "issueId": 1687,
+    "name": "Esha",
+    "username": "Esha98"
+}
+
```
Comment (Panquesito7) [inline profiles/esha98.json:7]:
```suggestion
}

```

File: profiles/esha98.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/51391473/215367152-fbb4084f-79a4-470b-a96a-1656f7a735dc.png",
+    "issueId": 1687,
+    "name": "Esha",
+    "username": "Esha98"
```
Comment (Panquesito7) [inline profiles/esha98.json:5]:
```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/215367152-fbb4084f-79a4-470b-a96a-1656f7a735dc.png",
  "issueId": 1687,
  "name": "Esha",
  "username": "Esha98"
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**